ui-tests/src/test/java/org/openmrs/reference/FindPatientByIdAndNameTest.java

```diff
@@ -1,78 +0,0 @@
-package org.openmrs.reference;
-
-import org.junit.Before;
-import org.junit.Test;
-import org.openmrs.reference.helper.TestPatient;
-import org.openmrs.reference.page.*;
-import org.openmrs.uitestframework.test.TestBase;
-import org.junit.*;
-import static org.junit.Assert.assertTrue;
-
-
-/**
- * Created by nata on 09.07.15.
- */
-
-public class FindPatientByIdAndNameTest extends TestBase {
-    private HomePage homePage;
-    private HeaderPage headerPage;
-    private FindPatientPage findPatientPage;
-    private TestPatient patient;
-    private RegistrationPage registrationPage;
-    private ClinicianFacingPatientDashboardPage patientDashboardPage;
-    private String id;
-
```

Comment:
There are currently 2 ways to find patients: ID and name, which includes given, middle and family. Currently, there's a test which searches patients using ID, FindPatientRecordTest. The file I've added, FindPatientByNameTest searches patients by its given, middle and family name, which covers all the use cases for finding patients.
The test I'm removing searches for patients using its id, which is already covered in FindPatientRecordTest, and is one of the running tests in CI.